task_functions/upload_db.py

```diff
@@ -0,0 +1,82 @@
+from datetime import datetime
+import re
+import logging
+
+from dag_datalake_sirene.config import (
+    MINIO_BUCKET,
+    MINIO_URL,
+    MINIO_USER,
+    MINIO_PASSWORD,
+    SIRENE_MINIO_DATA_PATH,
+    AIRFLOW_PREPROCESSING_DATA_DIR,
+    AIRFLOW_INDEXING_DATA_DIR,
+)
+from dag_datalake_sirene.utils.minio_helpers import (
+    get_files_from_prefix,
+    get_files,
+    send_files,
+)
+
+current_date = datetime.now().date()
+
+
+def send_to_minio(list_files):
+    send_files(
+        MINIO_URL=MINIO_URL,
+        MINIO_BUCKET=MINIO_BUCKET,
+        MINIO_USER=MINIO_USER,
+        MINIO_PASSWORD=MINIO_PASSWORD,
+        list_files=list_files,
+    )
+
+
+def upload_db_to_minio(**kwargs):
```

Comment:
@HAEKADI , you may also want to keep the latest days/weeks only in MinIO.

Reply:
Yes, I have another PR in the works that clean any databases that are older than 2 weeks and keeps the latest one.

Reply:
https://github.com/etalab/annuaire-entreprises-search-infra/pull/234